### ISSUE TITLE
Deprecation warning due to sunpy.net.vso.attrs

### ISSUE THREAD

**Issue by abhijeetmanhas** [sunpy/sunpy#3814]
I ran test for `test_fido` file and this warning pops up:
```
sunpy/net/tests/test_fido.py:268: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <[AttributeError("'Time' object has no attribute 'start'",) raised in repr()] Time object at 0x7f7eff4ebf98>
args = ('2011-06-07 06:33', '2011-06-07 06:33:08'), kwargs = {}, name = 'Time'

    def __init__(self, *args, **kwargs):
        name = type(self).__name__
        warnings.warn(f"sunpy.net.vso.attrs.{name} is deprecated, please use sunpy.net.attrs.{name}",
>                     SunpyDeprecationWarning)
E       sunpy.util.exceptions.SunpyDeprecationWarning: sunpy.net.vso.attrs.Time is deprecated, please use sunpy.net.attrs.Time

sunpy/net/vso/attrs.py:380: SunpyDeprecationWarning
```
I think it can be simply solved by replacing `sunpy.net.vso.attrs` by `sunpy.net.attrs`

**Comment by Cadair** [comment]
I *think* I have fixed this in #3770

**Comment by abhijeetmanhas** [comment]
@Cadair It is fixed, in #3770 but va is imported but not used, so it should be removed

**Comment by amogh-jrules** [comment]
Can I take this up?

**Comment by nabobalis** [comment]
It seems to be fixed in an open PR #3770. I would suggest looking for another issue.

**Comment by amogh-jrules** [comment]
> @Cadair It is fixed, in #3770 but va is imported but not used, so it should be removed

@nabobalis Oops, I saw this message a pr before you replied
